frontend/src/components/common/table/table.tsx

```diff
@@ -6,12 +7,20 @@ import styles from './styles.module.scss';
 type Props<Data extends Record<string, unknown>> = {
   columns: Column<Data>[];
   data: readonly Data[];
+  totalCount: number;
+  pageSize: number;
+  currentPage: number;
+  onPageChange: (newPage: number) => void;
```

Comment:
I am not sure, but I think that these props should not be passed into table component, but taken from the reducer (in my opinion it will be logically better incapsulation)

Reply:
@WilhelmWesser If you'd get pagination values from store inside table component - then it won't be reusable anymore

Reply:
@andanfe Oh......
Thank you, I missed that 😕